akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -84,14 +85,12 @@ class Http2ServerDemux extends GraphStage[BidiShape[Http2SubStream, FrameEvent,
       logic ⇒
 
       final case class SubStream(
-        streamId:                  Int,
-        state:                     StreamState,
-        outlet:                    Option[BufferedOutlet[ByteString]],
-        inlet:                     Option[SubSinkInlet[ByteString]],
-        initialOutboundWindowLeft: Long
-      ) {
-        var outboundWindowLeft = initialOutboundWindowLeft
```

Comment:
so... copy() initialized this with the initial outbound value again :O whereas we actually wanted to keep the value we accumulated here, since the WINDOW_UPDATEs were writing into here.